client/package.json

```diff
@@ -12,7 +12,9 @@
     "draft-js-plugins-editor": "^2.1.1",
     "draft-js-static-toolbar-plugin": "^3.0.0",
     "http-proxy-middleware": "^0.19.1",
+    "lodash.identity": "^4.4.0",
     "lodash.pick": "^4.4.0",
+    "lodash.pickby": "^4.4.0",
```

Comment:
We already use `lodash.pick` somewhere in the project, so no need to introduce redundant packages. Remove `lodash.pickby` and do the following instead.

```js
import pick from 'lodash.pick';
import identity from 'lodash.identity'

let filterObj = pick(objToFilter, identity)
```